es6-shim.js

```diff
@@ -255,12 +255,21 @@
   };
   var isArguments = isStandardArguments(arguments) ? isStandardArguments : isLegacyArguments;
 
+  var isNativeSymbol = function isNativeSymbol(value) {
+    return typeof globals.Symbol === 'function' && typeof value === 'symbol';
+  };
+
+  // idea from https://github.com/zloirock/core-js/blob/master/packages/core-js/modules/es.symbol.js#L88
+  var isNonNativeSymbol = function isNonNativeSymbol(value) {
+    return typeof globals.Symbol === 'function' && Object(value) instanceof globals.Symbol;
+  };
+
   var Type = {
     primitive: function (x) { return x === null || (typeof x !== 'function' && typeof x !== 'object'); },
     string: function (x) { return _toString(x) === '[object String]'; },
     regex: function (x) { return _toString(x) === '[object RegExp]'; },
     symbol: function (x) {
-      return typeof globals.Symbol === 'function' && typeof x === 'symbol';
+      return isNativeSymbol(x) || isNonNativeSymbol(x);
```

Comment:
i'd prefer to not even call `isNonNativeSymbol` in an environment with native symbols. Can we change this check so that `Type.symbol` itself is either `=== isNativeSymbol`, or `=== isNonNativeSymbol`, rather than having to do both checks at runtime?